proxy/json.go

```diff
@@ -45,6 +46,25 @@ func (j jsonObject) String(key string) (string, error) {
 	return result, nil
 }
 
+func (j jsonObject) Int(key string) (int, error) {
```

Comment:
Other than detecting when non-numeric characters have been supplied, do we gain anything from implementing this function?